doc/source/integer_na.rst

```diff
@@ -0,0 +1,84 @@
+.. currentmodule:: pandas
```

Comment:
not important, but the `.. currentmodule:: pandas` is also included in the `{{ header }}` (we only kept it in the `api.rst` because the autosummaries need it before the header is rendered)